doc/architecture/06runtime.md

```diff
@@ -15,6 +15,44 @@ sequenceDiagram
     Frontend->>Backend: place_plant (plant_ID, pos, date)
 ```
 
+## Onboarding
+
+```mermaid
+sequenceDiagram
+    actor User
+
+    User->>Permaplant: visit landing page
+
+    User->>Keycloak: self-registration
+    activate Keycloak
+
+
+    User->>Permaplant: visit public maps
+    activate Permaplant
+
+    Permaplant->>Nextcloud: use images etc.
+    activate Nextcloud
+    actor Admin
+
+    User->>Permaplant: membership application
+    Permaplant->>Admin: notification
+
+    alt accept
+        Admin->>Keycloak: change role to member
+    else accept
+        Admin->>Keycloak: remove user
```

Comment:
Why would the admin remove the user on an accept? Did you mean to title the else branch something else perhaps?
Besides that, I don't think we should completely remove a registered user when declining the membership application, since a Keycloak account seems to be neccessary for public maps.

Reply:
> Why would the admin remove the user on an accept? Did you mean to title the else branch something else perhaps?

Yes, this is the else branch of accept in mermaid syntax.

> Besides that, I don't think we should completely remove a registered user when declining the membership application, since a Keycloak account seems to be neccessary for public maps.

Yes, in general she shouldn't be removed. But on non-serious/spam-applications removal might be justified.

Reply:
I fixed this now. (Usually we will ask for a new application unless it is obviously spam)